WalletWasabi.Gui/Controls/WalletExplorer/ReceiveTabView.xaml

```diff
@@ -120,7 +122,7 @@
                       </Grid>
                     </Grid>
 
-                    <TextBlock Text="{Binding Label, ConverterParameter=11, Converter={StaticResource LurkingWifeModeStringConverter}, Mode=OneWay}" Grid.Column="1" />
+                    <controls:EditableTextBlock Text="{Binding Label, Mode=TwoWay}" InEditMode="{Binding InEditMode}"  Grid.Column="1" />
```

Comment:
Why remove the lurking wife mode?

Reply:
I was going to ask you about this.

If its in edit mode, then it would trigger a change in the label. My idea is probably to hide the editabletextblock when in lurking wife mode, or add a property like ReadOnlyMode to the editable textblock.

I plan to fix this before we merge.

Reply:
Back in now... that took some time to get right :)